desktop/coworker/server.py

```
@@ -631,6 +632,35 @@ async def _publish_live_events(events: list[dict[str, Any]]) -> None:
+        except ValueError:
+            return None
+        if session_id is None:
+            return None
+        row = app.state.store.index(session_id)
+        person = app.state.people.get(person_id)
+        if row is None or person is None:
+            return None
+        current = str(row.get("title") or "")
+        if current.startswith("Sourcing ·") or not current:
+            brief = build_brief(person, app.state.people.timeline(person_id))
+            safe = f"Sourcing · {str(brief['who'] or 'Person')}"
```

Ruff (0.16.2)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)